sys/include/net/netstats.h

```diff
@@ -18,13 +18,22 @@
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  */
 
+#include <stdint.h>
+
 #ifndef NETSTATS_H
 #define NETSTATS_H
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef enum {
+    NETSTATS_LAYER2     = 0x01,
+    NETSTATS_IPV6       = 0x02,
+    NETSTATS_ALL        = 0xFF
```

Comment:
Actually, I don't know whether @authmillenon's statement is correct or not,
but it's also written in our coding conventions:
[Thou shalt not use enums as flags](https://github.com/RIOT-OS/RIOT/wiki/Best-Practice-for-RIOT-Programming#donts)